tapgarden/planter.go

```diff
@@ -566,6 +566,17 @@ func (c *ChainPlanter) fundGenesisPsbt(ctx context.Context,
 			return nil, fmt.Errorf("unable to estimate fee: %w",
 				err)
 		}
+		minRelayFee, err := c.cfg.Wallet.MinRelayFee(ctx)
```

Comment:
As-is this won't catch the case of the user manually specifying a feerate below minrelayfee.

Also, in that case IMO you'd want to fail with an error message reporting the minrelayfee, instead of bumping the fee silently.

So I'd suggest moving this check after the switch instead of in the default case; then you can switch behavior based on `manualFeeRate != nil`.

Reply:
Correct. The issue was raised specifically in a scenario where the user did *not* specify a feerate. The reasoning for putting the check inside the switch was that if you do specify the feerate manually, you are fully responsible for it. You could *even* specify a feerate below minrelayfee.